packages/melos/lib/src/scripts.dart

```diff
@@ -129,15 +130,18 @@ class Script {
     required String workspacePath,
   }) {
     final scriptPath = 'scripts/$name';
-    String run;
+    var run = '';
```

Comment:
I would define this where the `if` is and have it as final too, and set the default value in the else case.

Reply:
What do you think about having it as nullable instead? So that it works in the same way as the `description`?

Reply:
Yes agree! I make it nullable instead as you mentioned in commit 5c19f3da0d50534a64ac035ef2e85e601e2a43a9